tests/conftest.py

```diff
@@ -148,14 +149,14 @@ def mock_is_item_available():
 def mock_can_be_requested(base_app):
     """Mock can_be_requested check."""
     base_app.config["CIRCULATION_POLICIES"]["request"]["can_request"] = \
-        lambda item_pid=None, document_pid=None: True
+        lambda loan=None: True
 
 
 @pytest.fixture()
 def mock_can_not_be_requested(base_app):
```

Comment:
Is still used as we can just change the `CIRCULATION_POLICIES_ALLOW_REQUEST` variable?